crates/wasi-preview1-component-adapter/src/macros.rs

```diff
@@ -28,14 +16,16 @@ macro_rules! eprintln {
     ($arg:tt) => {{
         // We have to expand string literals into byte arrays to prevent them
         // from getting statically initialized.
-        let message = byte_array_literals::str_nl!($arg);
-        $crate::macros::print(&message);
+        // We use `str` instead of `str_nl` because we're calling the logging
+        // API which expects lines.
+        let message = byte_array_literals::str!($arg);
+        $crate::macros::println(&message);
```

Comment:
We can revert the line-oriented changes and move this back to stderr, right?

Reply:
Yes, I've now reverted them.